sw/device/tests/meson.build

```diff
@@ -2,10 +2,48 @@
 # Licensed under the Apache License, Version 2.0, see LICENSE for details.
 # SPDX-License-Identifier: Apache-2.0
 
-subdir('dif')
+sw_tests = {}
 
+subdir('dif')
 subdir('aes')
 subdir('flash_ctrl')
 subdir('hmac')
 subdir('rv_timer')
 subdir('consecutive_irqs')
+
+foreach sw_test_name, sw_test_lib : sw_tests
+
+  foreach device_name, device_lib : sw_lib_arch_core_devices
```

Comment:
I made some comments yesterday about getting rid of `sw_lib_arch_core_devices`, but actually that was totally wrong. This looks fine to me.

That said, could you get rid of the empty lines between the two `foreach` lines and the two `endforeach` lines?